### DISCUSSION TITLE
Danger Zone

### DISCUSSION THREAD

**Discussion by 0rdinary-J0e** [supermerill/SuperSlicer#2479] [General]
I've recently Damaged my PEI Flex Plate. Could a feature in the future be added to limit prints from being placed on the gouged line. Possibly describing an area with standard coordinates.
   I've got some programming experience if I could get some ideas to get started I could attempt to contribute to the project.

**Comment by supermerill** [comment]
if you're using a printer with a bed texture, just open the texture in paint and paint the bad section in red. Then save it.
It should be in superslicer/rsources/profiles/vendor_name/ directory

IF you don't have a bed texture, you can create one from scratch in paint, and add it to your printer via the bed shape button.
